recipe/meta.yaml

```
@@ -22,22 +22,24 @@ requirements:
     - pip
   run:
     - python
-    - anaconda-cloud-cli
-    - anaconda-cloud-auth
+    - anaconda-cloud-auth 0.7.2
+    - anaconda-cli-base
+    - anaconda-client >=1.12.2
+    - conda
+    - backports
 
 test:
   imports:
     - anaconda_cloud
     - anaconda_cli_base
     - anaconda_cloud_auth
```

```suggestion
    - anaconda_cloud_auth
    - binstar_client
```
binstar_client is a name to be kept for backward compatibility

---

Ok, thanks for the context! I'll add a comment.